pkg/expr/classic/reduce.go

```diff
@@ -38,7 +38,7 @@ func (cr classicReducer) Reduce(series mathexp.Series) mathexp.Number {
 	value := float64(0)
 	allNull := true
 
-	vF := series.Frame.Fields[series.ValueIdx]
+	vF := series.Frame.Fields[1]
```

Comment:
The `1` here should maybe be mathexp.seriesTypeValIdx , but then it would need to be exported, which I don't want to do right now. will save this for future refactoring.